### DISCUSSION TITLE
Playlists with already linked guides?

### DISCUSSION THREAD

**Discussion by EvilCaptKirk** [iptv-org/community#562] [Q&A]
The info page for this repo says, "Playlists with already linked guides can be found in the [iptv-org/iptv](https://github.com/iptv-org/iptv) repository."

I've been to iptv-org and I've pulled their country-based playlist, but I don't find any programming guide info either inside or alongside the playlist.

Am I missing something?

**Comment by BellezaEmporium** [comment]
That used to be the case before, but now since GH Actions are disabled onto the EPG repo, we cannot do such stuff. It's some old formulation that needs to be replaced.

**Comment by freearhey** [comment]
This sentence has been removed from the repository description in https://github.com/iptv-org/epg/commit/3e36c64ef9ac6c6b934db50b794d345e4d63737e
